extensions/soundboard/src/utils.ts

```diff
@@ -1,28 +1,66 @@
 import { exec } from "child_process";
 import { Item } from "./types";
 import { getItem, getItems, saveItems } from "./storage";
-import { launchCommand, LaunchType } from "@raycast/api";
+import { environment, launchCommand, LaunchType, updateCommandMetadata } from "@raycast/api";
 
 export const playSoundFromIndex = async (index: number) => {
   const sound = await getItem(index);
   if (sound) {
-    await playFile(sound);
+    if (environment.launchType === LaunchType.UserInitiated) {
+      await playFile(sound);
+    }
+
+    await updateCommandMetadata({ subtitle: `Sound Name: ${sound.title}` });
     return;
   }
 
-  launchCommand({ name: "index", type: LaunchType.UserInitiated, context: { index: index } });
+  if (environment.launchType === LaunchType.UserInitiated) {
+    launchCommand({ name: "index", type: LaunchType.UserInitiated, context: { index: index } });
+  }
+
+  await updateCommandMetadata({ subtitle: "Soundboard" });
 };
 
 export const playFile = async (item: Item) => {
   exec(`afplay ${item.path} && $$`);
 };
 
+export const updateFavoriteSubtitles = async (item: Item) => {
+  if (item.favourite !== "0") {
+    await launchCommand({ name: `favorite${item.favourite}`, type: LaunchType.Background });
```

Comment:
I know it's not possible, but it'd be nice if `launchCommand` supported updating the `title` as well. That way, it would help with the initial UX of which commands can/cannot play a sound. 

- "Set Favourite": when none is set
- "Play Favourite": when it's set (it will actually play it in this case)

<img width="795" alt="(ಠ ›ಠ) — @peduarte — C8eyoS2R000045@2x" src="https://user-images.githubusercontent.com/372831/202390011-f183183a-49a2-436d-90a8-57c4104bd547.png">